### PR TITLE
Add observability section to main page

### PR DESCRIPTION
**Pull request by lucperkins**
Not sure if this is desired or the right way to go about it, but I think it might be nice to give a breakdown of the different domains of observability. If this content is better left for a later date or a separate doc, I'm cool with that.

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -6,6 +6,19 @@ disableKinds = ["taxonomy", "taxonomyTerm"]
 tagline = "Effective observability requires high-quality telemetry"
 sub_tagline = "**OpenTelemetry** makes robust, portable telemetry a built-in feature of cloud-native software."
 font_awesome_version = "5.8.1"
+observability = """
+The term [**observability**](https://en.wikipedia.org/wiki/Observability) stems from the discipline of [control theory](https://en.wikipedia.org/wiki/Control_theory) and refers to how well a system can be understood on the basis of the **telemetry** that it produces.
+
+In software, observability typically refers to telemetry produced by **services** and is divided into three major verticals:
+
+* [**Tracing**](https://opentracing.io/docs/overview/what-is-tracing), aka **distrbuted tracing**, provides insight into the full lifecycles, aka *traces*, of requests to the system, allowing you to pinpoint failures and performance issues.  
+* [**Metrics**](https://opencensus.io/stats) provide quantitative information about processes running inside the system, including counters, gauges, and histograms. This domain also encompasses querying that data and configuring automated responses to quantitatively specified abormal states.
```
Comment (SergeyKanzhelev) [inline config.toml]: > This domain also encompasses querying that data and configuring automated responses to quantitatively specified abormal states.

this goes into the prometheus territory. I'm not sure it's an OpenTelemetry charter

File: config.toml [hunk 2]
```diff
@@ -6,6 +6,19 @@ disableKinds = ["taxonomy", "taxonomyTerm"]
...
+* [**Logging**](https://en.wikipedia.org/wiki/Log_file) provides insight into application-specific messages emitted by processes.
+
```
Comment (SergeyKanzhelev) [inline config.toml]: I'd add a sentence like:

Those verticals are interconnected. **Metrics** can be used to pinpoint subset of misbehaving **traces**. **Logs** associated with those traces would help to find the root cause of this behavior. And than new **metrics** can be configured based on this discovery to catch this issue earlier the next time.

File: config.toml [hunk 3]
```diff
@@ -6,6 +6,19 @@ disableKinds = ["taxonomy", "taxonomyTerm"]
...
+OpenTelemetry is an effort to combine all three verticals into a single set of system components and language-specific telemetry libraries. It is meant to replace both the [OpenTracing](https://opentracing.io) project, which focused exclusively on tracing, and the [OpenCensus](https://opencensus.io) project, which focused on tracing and metrics.
```
Comment (SergeyKanzhelev) [inline config.toml]: I'm not sure how much OpenTracing emphasized logs support (see https://opentracing.io/docs/overview/tags-logs-baggage/). I understand there were criticism of OpenTracing approach to solve logging scenarios, so not sure how much it needs advertisement. And OpenCensus has the similar concept, but it was never intended nor called logs.

Anyway, just something that @bhs may want to pay attention to.

File: config.toml (hunk 1 above)
Comment (mtwo) [reply config.toml]: +1

I recommend removing the sentence that Sergey highlighted

File: config.toml (hunk 2 above)
Comment (mtwo) [reply config.toml]: +1

File: config.toml [hunk 4]
```diff
@@ -6,6 +6,19 @@ disableKinds = ["taxonomy", "taxonomyTerm"]
...
+
+Intially, OpenTelemetry will not support logging. This vertical will be incorporated over time.
```
Comment (mtwo) [inline config.toml]:
```suggestion
OpenTelemetry will not initially support logging, though we aim to incorporate this over time.
```

File: config.toml (hunk 2 above)
Comment (lucperkins) [reply config.toml]: I love this

**Comment by lucperkins** [comment]
Okay, I've made requested changes. If someone could give this one last look-over, that'd be great.

**Pull request merged**